templates/includes/sidebar.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block sidebar %}

```

Comment:
when we use include we don't need to extend base.html and make new blocks. Just write needed HTML code and it's gonna work

Reply:
Thanks for the advice

Reply:
Fixed, but i can`t change status in table